bioimageio/weight_converter/torch/torchscript.py

```diff
@@ -65,9 +65,9 @@ def main():
     parser.add_argument("--tracing", "-t", default=1, type=int)
 
     args = parser.parse_args()
-    return convert_weights_to_torchscript(os.path.abspath(args.model),
-                                          args.output,
-                                          bool(args.tracing))
+    return convert_weights_to_pytorch_script(os.path.abspath(args.model),
```

Comment:
A quick jump-in comment, isn't `TorchScript ` a fixed term?

Reply:
Yeah, I was also wondering about this, but we call the weight format `pytorch_script`, so I wanted to make it consistent here.

Reply:
Aha, so it might be better to rename the format instead?

Reply:
We could set it onto the list of potential 0.4 changes.